tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py

```diff
@@ -34,6 +34,12 @@ def test_browser_bookmark(self):
         bookmark.type_bookmark_title(self.bookmark_title)
         bookmark.tap_add_bookmark_to_home_screen_dialog_button()
 
+        # tap menu button to test to see if the add to homescreen button is removed
+        browser.switch_to_chrome()
+        browser.tap_menu_button()
+        self._add_to_home=bookmark.is_add_to_home_button_not_in_dialog()
```

Comment:
Agree, you should use <code>bookmark.is_add_to_home_button_not_in_dialog</code> method directly in the assert.